routes/organization.js

```diff
@@ -7,9 +7,9 @@ const organizationValidator = require('../validations/organizationValidator');
 const awsImageUploader = require('../utils/awsImageUploader');
 
 // Organization update
-router.post('/public', authAdmin, organizationValidator, awsImageUploader, organizationUpdate);
+/* router.post('/public', authAdmin, organizationValidator, awsImageUploader, organizationUpdate);
 
 router.post('/newOrg', [], create);
 router.get('/public', [], getData);
```

Comment:
Esta ruta es requerida

> desde [Jira](https://alkemy-labs.atlassian.net/browse/OT176-23) -... Se debe realizar una petición GET hacia ruta /organization/public

```suggestion
*/
router.get('/public', getData);
/*
```